common/content/statusline.js

```diff
@@ -127,17 +127,13 @@ const StatusLine = Module("statusline", {
 
     // set the visibility of the statusline
     setVisibility: function (request) {
-        if ( typeof this.setVisibility.currVisibility == 'undefined' ) {
-            this.setVisibility.currVisibility = true;
-            this.setVisibility.prevVisibility = true;
-            this.setVisibility.contentSeparator
-                = highlight.get('ContentSeparator').value;
-
-            // kinds of requests
-            this.setVisibility.MODE_ON  = 0;      // commandline active
```

Comment:
unfortunately, I think we have to keep these around for a bit longer so that we don't break anyone who has already used them. The NEWS file only mentioned `TOGGLE`, so, just keeping that one is good by me.
Is `UPDATE` supposed to be `TOGGLE`? If so, I think the name `TOGGLE` is a better representation of what is happening.